apps/dashboard/src/components/subscribers/subscriber-overview-form.tsx

```diff
@@ -0,0 +1,298 @@
+import { formatDateSimple } from '@/utils/format-date';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { loadLanguage } from '@uiw/codemirror-extensions-langs';
+import { useForm } from 'react-hook-form';
+import { RiDeleteBin2Line } from 'react-icons/ri';
+import { z } from 'zod';
+import { Button } from '../primitives/button';
+import { Editor } from '../primitives/editor';
+import { Form, FormControl, FormField, FormItem, FormLabel, FormMessage } from '../primitives/form/form';
+import { Input, InputRoot } from '../primitives/input';
+import { PhoneInput } from '../primitives/phone-input';
+import { Separator } from '../primitives/separator';
+import { SubscriberFormSchema } from './schema';
+import { TimezoneSelect } from './timezone-select';
+import { Avatar, AvatarFallback, AvatarImage } from '../primitives/avatar';
+import { Tooltip, TooltipContent, TooltipTrigger } from '../primitives/tooltip';
+import { Link } from 'react-router-dom';
+import { usePatchSubscriber } from '@/hooks/use-patch-subscriber';
+import { showSuccessToast } from '../primitives/sonner-helpers';
+import { SubscriberResponseDto } from '@novu/api/models/components';
+import { CopyButton } from '../primitives/copy-button';
+import { SubscriberOverviewSkeleton } from './subscriber-overview-skeleton';
+
+const extensions = [loadLanguage('json')?.extension ?? []];
+const basicSetup = { lineNumbers: true, defaultKeymap: true };
+
+export default function SubscriberOverviewForm({
```

Comment:
If not mistaken we prefer named exports also for internal (Dashboard) DX
```suggestion
export function SubscriberOverviewForm({
```